tests/Unit/Evolution/Systems/Cce/Test_Equations.cpp

```diff
@@ -0,0 +1,202 @@
+  static std::string name() noexcept {
+    return "compute_integrand_for_q_pole_part";
+  }
+};
+template <>
+struct python_function_for_bondi_integrand<
+    Tags::RegularIntegrand<Tags::BondiQ>> {
+  static std::string name() noexcept {
+    return "compute_integrand_for_q_regular_part";
+  }
+};
+template <>
+struct python_function_for_bondi_integrand<Tags::Integrand<Tags::BondiU>> {
+  static std::string name() noexcept { return "compute_integrand_for_u"; }
+};
+template <>
+struct python_function_for_bondi_integrand<
+    Tags::PoleOfIntegrand<Tags::BondiW>> {
+  static std::string name() noexcept {
+    return "compute_integrand_for_w_pole_part";
+  }
+};
+template <>
+struct python_function_for_bondi_integrand<
+    Tags::RegularIntegrand<Tags::BondiW>> {
+  static std::string name() noexcept {
+    return "compute_integrand_for_w_regular_part";
+  }
+};
+template <>
+struct python_function_for_bondi_integrand<
+    Tags::PoleOfIntegrand<Tags::BondiH>> {
+  static std::string name() noexcept {
+    return "compute_integrand_for_h_pole_part";
+  }
+};
+template <>
+struct python_function_for_bondi_integrand<
+    Tags::RegularIntegrand<Tags::BondiH>> {
+  static std::string name() noexcept {
+    return "compute_integrand_for_h_regular_part";
+  }
+};
+template <>
+struct python_function_for_bondi_integrand<Tags::LinearFactor<Tags::BondiH>> {
+  static std::string name() noexcept { return "compute_linear_operator_for_h"; }
+};
+template <>
+struct python_function_for_bondi_integrand<
+    Tags::LinearFactorForConjugate<Tags::BondiH>> {
+  static std::string name() noexcept {
+    return "compute_linear_operator_for_conjugate_h";
+  }
+};
+
+SPECTRE_TEST_CASE("Unit.Evolution.Systems.Cce.Equations", "[Unit][Evolution]") {
+  pypp::SetupLocalPythonEnvironment local_python_env{"Evolution/Systems/Cce/"};
+
+  using all_bondi_tags = tmpl::list<Tags::Beta, Tags::BondiQ, Tags::BondiU,
+                                    Tags::BondiW, Tags::BondiH>;
+  tmpl::for_each<all_bondi_tags>([](auto x) {
+    using bondi_tag = typename decltype(x)::type;
+    tmpl::for_each<integrand_terms_to_compute_for_bondi_variable<bondi_tag>>(
+        [](auto y) {
+          using bondi_integrand_tag = typename decltype(y)::type;
+          using bondi_integrand_argument_list =
+              all_arguments_for_integrand<bondi_integrand_tag>;
+          // We check the equations with the variables interface, as it is easy
+          // to generate a set of Variables with the appropriate type lists,
+          // which are then used to create wrappers that can be compared to Pypp
+          // equations without explicitly enumerating the long argument lists in
+          // this test.
+          forward_to_pypp_with<ComputeBondiIntegrand<bondi_integrand_tag>,
+                               bondi_integrand_argument_list,
+                               bondi_integrand_tag, 5, ComplexDataVector>(
+              std::make_index_sequence<5>{},
```

Comment:
Is the `5` the number of `all_bondi_tags`? Perhaps add a comment or even write as `tmp::size<all_bondi_tags>`.